gloo/docs/installation/quick_start.md

```diff
@@ -1,198 +1,325 @@
 ---
-title: Open Source Quick Start
-weight: -1
+title: Kubernetes - Installing Gloo
+weight: 2
 ---
 
-## Installing Gloo
+{{% notice note %}}
+To install Gloo Enterprise you need a License Key. If you don't have one, go to **https://solo.io/glooe-trial** and
```

Comment:
Im thinking we should have a completely separate section or page for Enterprise stuff. We don't want to cloud the open-source/enterprise stuff too much and confuse people into thinking they need license keys or anything.